pkg/util/runtime/fuzz.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC All Rights Reserved.
```

Comment:
I'm not sure that `runtime` is a good package for this.

Maybe in /tests/fuzz ?

Reply:
It has been moved to `/pkg/testing/fuzzing`, or would you prefer them placed in `/test/fuzz`?